Tempura/UITests/UIView+snapshot.swift

```diff
@@ -47,6 +47,10 @@ extension UIView {
       window?.addSubview(self)
       removeFromSuperview = true
     }
+
+    if let vc = viewToWaitFor?.next as? UIViewController {
```

Comment:
You should not assume the VC we want to configure is the `nextResponder` of the View, some views could have set a custom NSObject as their nextResponder.